scipy/stats/_stats_py.py

```diff
@@ -7169,51 +7169,50 @@ def ks_1samp(x, cdf, args=(), alternative='two-sided', mode='auto'):
-
-    *Test against one-sided alternative hypothesis*
+    Suppose we wish to assess whether data are distributed according to the
+    standard normal distribution.
+    We choose a confidence level of 95%; that is, we will reject the null
+    hypothesis in favor of the alternative if the p-value is less than 0.05.
 
-    Shift distribution to larger values, so that `` CDF(x) < norm.cdf(x)``:
+    When testing uniformly distributed data, we would expect the
+    null hypothesis to be rejected.
 
-    >>> x = stats.norm.rvs(loc=0.2, size=100, random_state=rng)
-    >>> stats.ks_1samp(x, stats.norm.cdf, alternative='less')
-    KstestResult(statistic=0.100203351482..., pvalue=0.125544644447...)
+    >>> stats.ks_1samp(stats.uniform.rvs(size=100, random_state=rng),
+    ...                stats.norm.cdf)
+    KstestResult(statistic=0.5001899973268688, pvalue=1.1616392184763533e-23)
 
-    Reject null hypothesis in favor of alternative hypothesis: less
+    Indeed, the p-value is lower than our threshold of 0.05, so we reject the
+    null hypothesis that the data are normally distributed.
 
-    >>> stats.ks_1samp(x, stats.norm.cdf, alternative='greater')
-    KstestResult(statistic=0.018749806388..., pvalue=0.920581859791...)
-
-    Reject null hypothesis in favor of alternative hypothesis: greater
+    When testing random variates from the standard normal distribution, we
+    expect the data to be consistent with the null hypothesis most of the time.
 
+    >>> x = stats.norm.rvs(size=100, random_state=rng)
     >>> stats.ks_1samp(x, stats.norm.cdf)
-    KstestResult(statistic=0.100203351482..., pvalue=0.250616879765...)
+    KstestResult(statistic=0.05345882212970396, pvalue=0.9227159037744717)
 
-    Don't reject null hypothesis in favor of alternative hypothesis: two-sided
+    As expected, the p-value of 0.92 is not below our threshold of 0.05, so
+    we cannot reject the null hypothesis.
 
-    *Testing t distributed random variables against normal distribution*
+    Suppose, however, that the location of the normally distributed random
+    variates is shifted toward larger values. In this case, the CDF of the
+    random variates tends to be *less* than the CDF of the standard normal.
+    Therefore, we would expect the null hypothesis to be rejected with
+    ``alternative='less'``:
 
-    With 100 degrees of freedom the t distribution looks close to the normal
-    distribution, and the K-S test does not reject the hypothesis that the
-    sample came from the normal distribution:
+    >>> x = x + 0.5
+    >>> stats.ks_1samp(x, stats.norm.cdf, alternative='less')
+    KstestResult(statistic=0.21707976606981766, pvalue=6.435039066115298e-05)
 
-    >>> stats.ks_1samp(stats.t.rvs(100,size=100, random_state=rng),
-    ...                stats.norm.cdf)
-    KstestResult(statistic=0.064273776544..., pvalue=0.778737758305...)
+    and with ``alternative='two-sided'``:
```

Comment:
How about
```suggestion
    and indeed, with a p-value that is nearly zero, we reject the null
    hypothesis that the data are distributed according to the standard
    normal.
```
and we remove the rest?